Odata-docs/TOC.yml

```diff
@@ -167,6 +167,8 @@
     items:
```

Comment:
there are build errors and warnings that need to be resolved. can you please look into it.

Reply:
> there are build errors and warnings that need to be resolved. can you please look into it.

[Link to build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C17%5C2ea817ce-bd30-41f2-f2eb-9eb93b4d61ef%5CPullRequest%5C201912170118339753-83%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rQ4Jyim8JUiGXuUB9Sk5IJlgcKIPYEn55%2FGE4dcoJ%2Fk%3D&st=2019-12-17T01%3A14%3A45Z&se=2020-01-17T01%3A19%3A45Z&sp=r)

There are no build `Warning` or `Suggestion` from the files in this PR. I fixed all the build errors in your repro in my previous PR. Most of the warnings are `Missing attribute: author. Add the current author's GitHub ID.`